packages/node/src/transports/base.ts

```diff
@@ -70,6 +70,26 @@ export abstract class BaseTransport implements Transport {
     return this._buffer.drain(timeout);
   }
 
+  /** Extracts proxy settings from client options and env variables */
```

Comment:
Should we include in the docs that if HTTP proxy is not found, it looks for the HTTPS proxy?

Reply:
Updated